src/rust/ide/lib/enso-protocol/src/project_manager.rs

```diff
@@ -38,7 +38,9 @@ trait API {
     /// Request the project manager to open a specified project. This operation also
     /// includes spawning an instance of the language server open on the specified project.
```

Comment:
It might be good to add mention to the doc string when exactly a `MissingComponentAction` might be triggered.